docs/advanced_topics/api/v2/usage.md

```diff
@@ -0,0 +1,647 @@
+following endpoints:
+
+-   Pages `/api/v2/pages/`
+-   Images `/api/v2/images/`
+-   Documents `/api/v2/documents/`
+
+> The available endpoints and their URLs may vary from site to site, depending
+> on how the API has been configured.
+
+### Example response
+
+Each response contains the list of items (`items`) and the total count
+(`meta.total_count`). The total count is irrespective of pagination.
+
+```
+GET /api/v2/endpoint_name/
+
+HTTP 200 OK
+Content-Type: application/json
+
+{
+    "meta": {
+        "total_count": "total number of results"
+    },
+    "items": [
+        {
+            "id": 1,
+            "meta": {
+                "type": "app_name.ModelName",
+                "detail_url": "http://api.example.com/api/v2/endpoint_name/1/"
+            },
+            "field": "value"
+        },
+        {
+            "id": 2,
+            "meta": {
+                "type": "app_name.ModelName",
+                "detail_url": "http://api.example.com/api/v2/endpoint_name/2/"
+            },
+            "field": "different value"
+        }
+    ]
+}
+```
+
+### Custom page fields in the API
+
+Wagtail sites contain many page types, each with their own set of fields. The
+`pages` endpoint will only expose the common fields by default (such as
+`title` and `slug`).
+
+To access custom page fields with the API, select the page type with the
+`?type` parameter. This will filter the results to only include pages of that
+type but will also make all the exported custom fields for that type available
+in the API.
+
+For example, to access the `published_date`, `body` and `authors` fields
+on the `blog.BlogPage` model in the [:ref:\`configuration docs <apiv2_page_fields_configuration>\`:]()
+
+```
+GET /api/v2/pages/?type=blog.BlogPage&fields=published_date,body,authors(name)
+
+HTTP 200 OK
+Content-Type: application/json
+
+{
+    "meta": {
+        "total_count": 10
+    },
+    "items": [
+        {
+            "id": 1,
+            "meta": {
+                "type": "blog.BlogPage",
+                "detail_url": "http://api.example.com/api/v2/pages/1/",
+                "html_url": "http://www.example.com/blog/my-blog-post/",
+                "slug": "my-blog-post",
+                "first_published_at": "2016-08-30T16:52:00Z"
+            },
+            "title": "Test blog post",
+            "published_date": "2016-08-30",
+            "authors": [
+                {
+                    "id": 1,
+                    "meta": {
+                        "type": "blog.BlogPageAuthor",
+                    },
+                    "name": "Karl Hobley"
+                }
+            ]
+        },
+
+        ...
+    ]
+}
+```
+
+> Only fields that have been explicitly exported by the developer may be used
+> in the API. This is done by adding a `api_fields` attribute to the page
+> model. You can read about configuration [:ref:\`here <apiv2_page_fields_configuration>\`]().
+
```

Comment:
Same here with the `>` - likely not needed.

Reply:
@lb- and @zerolab, thanks for reviewing!
I'll make the changes you suggested! 😄